toolbox

```diff
@@ -737,7 +737,7 @@ list_images()
     fi
 
     if [ "$output" != "" ]; then
-        echo -e "${LBC}Images created by toolbox${NC}"
+        printf "%sImages created by toolbox%s" "${LBC}" "${NC}"
```

Comment:
Sadly, we can't use `%s` in the format string to escape `LBC`, `NC`, etc.. :(

The problem is that those variables have the terminal escape sequences to change the foreground colour. Unless those variables are expanded inside the format string itself, those sequences won't be interpreted as control characters and will instead be printed as is.

If we keep the variable expansion inside the format string, then we run into [SC2059](https://github.com/koalaman/shellcheck/wiki/SC2059), which needs to be supressed. :/

Anyway, this was addressed in commit 189f366deafed1a225003a2f14f969540b60dbc8 from PR #56.

Reply:
Rebased on top of master.